ipv8/REST/attestation_endpoint.py

```diff
@@ -21,8 +21,12 @@ class AttestationEndpoint(resource.Resource):
     def __init__(self, session):
         resource.Resource.__init__(self)
         self.session = session
-        attestation_overlays = [overlay for overlay in session.overlays if isinstance(overlay, AttestationCommunity)]
-        identity_overlays = [overlay for overlay in session.overlays if isinstance(overlay, IdentityCommunity)]
+
+        attestation_overlays = [overlay for overlay in session.overlays if overlay.__class__.__name__ ==
```

Comment:
Could you revert these changes (go back to `isinstance`)? I'll try to find out what is causing it not to work.

Reply:
Sure!